deploy/base/deployment.yaml

```diff
@@ -146,6 +146,11 @@ spec:
                   key: mixer_hash.txt
           ports:
             - containerPort: 12345
+          startupProbe:
+            exec:
+              command: ["/bin/grpc_health_probe", "-addr=:12345"]
+            failureThreshold: 30
```

Comment:
Can you try to deploy this to dev. I tried this 2 years back but was not supported in old GKE cluster. I assume it should work now (cluster in 1.21) but should confirm.

Reply:
Also can you add startupProbe to the website container below as well?